src/main/webapp/WEB-INF/tags/instructor/results/sessionPanel.tag

```diff
@@ -21,8 +21,14 @@
             <label class="col-lg-4 control-label">Session:</label>
             <div class="col-lg-8">
               <p class="form-control-static">${sessionPanel.feedbackSessionName}
-                <c:if test="${not empty sessionPanel.editLink}">
-                  <a href="${sessionPanel.editLink}">[Edit]</a>
+                <c:if test="${not empty sessionPanel.editLink}">&nbsp
+                  <a href="${sessionPanel.editLink}"
+                     id="edit_icon_button"
```

Comment:
For line 26-29 follow 4 space indentation as shown here in the image

https://github.com/TEAMMATES/teammates/pull/8923#discussion_r199420404